psydac/feec/multipatch/plotting_utilities.py

```diff
@@ -0,0 +1,304 @@
+# coding: utf-8
+
+# Conga operators on piecewise (broken) de Rham sequences
```

Comment:
As CONGA is not part of this PR, I think that this comment should be updated

Reply:
yes, I deleted it in [9b8a549](https://github.com/pyccel/psydac/pull/160/commits/9b8a549c956b3c768890aa18b15c3979fa48e517)